apps/vanilla/src/pages/exercise2/scripts/const.ts

```diff
@@ -0,0 +1,2 @@
+/** Required number of points. */
+export const winningPoints = 21;
```

Comment:
Let's stick to our [naming convention](https://wiki.saritasa.rocks/frontend/languages/ts-js/naming#naming-convention)

Reply:
Do you mean something like this?
```suggestion
export const pointsNumberToWin = 21;
```

Reply:
No, I mean something like `WINNING_POINTS` :)